ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -53,7 +53,6 @@ tests:
     make lint
   container:
     from: assisted-test-infra
-  optional: true
```

Comment:
you need to update the job file as well

Reply:
yeah they still allow this for some reason
might be valuable to tell them on #forum-testplatform